code/game/objects/items/RCL.dm

```diff
@@ -221,18 +221,18 @@
 
 /obj/item/twohanded/rcl/proc/wiringGuiGenerateChoices(mob/user)
 	var/fromdir = 0
-	var/obj/structure/cable/linkingCable = findLinkingCable(user)
+	var/obj/structure/pipe_cleaner/linkingCable = findLinkingCable(user)
 	if(linkingCable)
 		fromdir = linkingCable.d2
 
 	var/list/wiredirs = list("1","5","4","6","2","10","8","9")
 	for(var/icondir in wiredirs)
 		var/dirnum = text2num(icondir)
-		var/cablesuffix = "[min(fromdir,dirnum)]-[max(fromdir,dirnum)]"
-		if(fromdir == dirnum) //cables can't loop back on themselves
-			cablesuffix = "invalid"
-		var/image/img = image(icon = 'icons/mob/radial.dmi', icon_state = "cable_[cablesuffix]")
-		img.color = GLOB.cable_colors[colors[current_color_index]]
+		var/pipe_cleanersuffix = "[min(fromdir,dirnum)]-[max(fromdir,dirnum)]"
+		if(fromdir == dirnum) //pipe_cleaners can't loop back on themselves
+			pipe_cleanersuffix = "invalid"
+		var/image/img = image(icon = 'icons/mob/radial.dmi', icon_state = "pipe_cleaner_[pipe_cleanersuffix]")
```

Comment:
You'll need to either rename the icon states in radial.dmi to match `pipe_cleaner_0-1` and so on or leave this as `cable_[pipe_cleanersuffix]`